configure.ac

```diff
@@ -134,11 +134,13 @@ AS_IF([test x"$with_rpc" != "xno"], [
         have_rpc=maybe
         save_CPPFLAGS=$CPPFLAGS
         CPPFLAGS="$CPPFLAGS $RPC_CFLAGS"
-        AC_CHECK_HEADERS([rpc/rpc.h],, [have_rpc=no])
+        dnl XXX The fourth argument to AC_CHECK_HEADERS is required for autoconf
+        dnl     2.69 to enable modern (compiler-based) behavior; don't remove.
```

Comment:
this is too ambiguous for people to understand or take action on.  make it clear what it's doing and when it can be removed.

```
dnl The 4th arg is required with autoconf-2.69 to disable preprocessor checking.
dnl This forces behavior adopted by autoconf-2.70+.
dnl TODO: Remove this check when we AC_PREREQ 2.70+.
```

Reply:
Fixed :).